spec/models/course_lesson_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe CourseLesson, type: :model do
+  it "can be created" do
+    expect {
+      CourseLesson.create(
+        title: "Test Course lesson",
+        content: "No content",
+        course_module: FactoryBot.create(:course_module),
+      )
+    }.to change { CourseLesson.count }.by(1)
+  end
+
+  describe "associations" do
+    it { is_expected.to belong_to(:course_module) }
```

Comment:
Could we test these associations as well: has_one :next_lesson, class_name: "CourseLesson"
  has_one :previous_lesson, class_name: "CourseLesson"